tinyagentos/app.py

```
@@ -1083,6 +1083,12 @@ async def _popularity_warm_loop(app: FastAPI) -> None:
         )
         # Start the live backend catalog — everything that asks "what's
         # available?" reads from this rather than the filesystem.
+        # tsk-xjwolt: start() does NOT block on the first probe anymore.
+        # Unreachable backends would otherwise stack their connect timeouts
+        # before :6969 can accept a single request. Probes run in the
+        # background; the lifecycle-state reconcile below fires on the
+        # first probe pass via a one-shot subscriber.
+        backend_catalog.subscribe(_reconcile_auto_manage_lifecycle)
```

Ruff (0.16.3)

[error] Undefined name `_reconcile_auto_manage_lifecycle`

(F821)